debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorIT.java

```diff
@@ -3486,6 +3487,76 @@ public void shouldStreamTruncateEventWhenLobIsEnabled() throws Exception {
         }
     }
 
+    @Test
+    @FixFor("DBZ-4963")
+    public void shouldRestartLogMiningSessionAfterMaxSessionElapses() throws Exception {
+        TestHelper.dropTable(connection, "dbz4963");
+        try {
+            connection.execute("CREATE TABLE dbz4963 (id numeric(9,0) primary key, data varchar2(50))");
+            TestHelper.streamTable(connection, "dbz4963");
+
+            Configuration config = TestHelper.defaultConfig()
+                    .with(OracleConnectorConfig.TABLE_INCLUDE_LIST, "DEBEZIUM\\.DBZ4963")
+                    .with(OracleConnectorConfig.LOG_MINING_SESSION_MAX_MS, 10_000L)
+                    .build();
+
+            LogInterceptor logInterceptor = new LogInterceptor(LogMinerStreamingChangeEventSource.class);
+            start(OracleConnector.class, config);
+            assertConnectorIsRunning();
+
+            waitForStreamingRunning(TestHelper.CONNECTOR_NAME, TestHelper.SERVER_NAME);
+
+            // Wait at most 60 seconds until we get notified in the logs that maximum session had exceeded.
+            Awaitility.await()
+                    .atMost(60, TimeUnit.SECONDS)
+                    .until(() -> logInterceptor.containsMessage("LogMiner session has exceeded maximum session time"));
+
+            stopConnector();
+        }
+        finally {
+            TestHelper.dropTable(connection, "dbz4963");
+        }
+    }
+
+    @Test
+    @FixFor("DBZ-4963")
+    public void shouldNotRestartLogMiningSessionWithMaxSessionZero() throws Exception {
+        TestHelper.dropTable(connection, "dbz4963");
+        try {
+            connection.execute("CREATE TABLE dbz4963 (id numeric(9,0) primary key, data varchar2(50))");
+            TestHelper.streamTable(connection, "dbz4963");
+
+            // default max session is 0L
+            Configuration config = TestHelper.defaultConfig()
+                    .with(OracleConnectorConfig.TABLE_INCLUDE_LIST, "DEBEZIUM\\.DBZ4963")
+                    .build();
+
+            LogInterceptor logInterceptor = new LogInterceptor(LogMinerStreamingChangeEventSource.class);
+            start(OracleConnector.class, config);
+            assertConnectorIsRunning();
+
+            waitForStreamingRunning(TestHelper.CONNECTOR_NAME, TestHelper.SERVER_NAME);
+
+            try {
+                // Wait at most 60 seconds for potential message
+                // The message should never be written here and so a ConditionTimeoutException should be thrown.
+                Awaitility.await()
```

Comment:
This is going to add 1 minute to the testsuite runtime, I am not really sure if it is worthy to have the test enabled by default.

Reply:
I will disable the test by default.